CONTRIBUTING.md

```diff
@@ -52,3 +52,43 @@ your current agreements on file or to sign a new one.
 You generally only need to submit a CLA once, so if you've already submitted one
 (even if it was for a different project), you probably don't need to do it
 again.
+
+## Maintainer's Guide
+
+`dep` has subsystem maintainers; this guide is intended for them in performing their work as a maintainer.
+
+### General guidelines
+
+* _Be kind, respectful, and inclusive_. Really live [that CoC](https://github.com/golang/dep/blob/master/CODE_OF_CONDUCT.md). We've developed a reputation as one of the most welcoming and supportive project environments in the Go community, and we want to keep that up!
+* The lines of responsibility between maintainership areas can be fuzzy. Get to know your fellow maintainers - it's important to work _with_ them when an issue falls in this grey area.
+* Remember, the long-term goal of `dep` is to disappear into the `go` toolchain. That's going to be a challenging process, no matter what. Minimizing that eventual difficulty should be a guiding light for all your decisions today.
```

Comment:
Yeah, I was musing on the impracticality of this advice while writing it. I'll probably take it out for now, until I can figure out how to deal with it better.